### ISSUE TITLE
Show Activity Groups in Assess tab of the UI

### ISSUE THREAD

**Issue by avaidyam** [BIDMCDigitalPsychiatry/LAMP-platform#4]
Basecamp Conversation
---
> **Previous Title:** Batch scheduling feeds and daily activities in Assess section related to the batch(lamp.group)
> @sarithapillai8: Effort estimation - 3days including development and QA

**Comment by sarithapillai8** [comment]
@avaidyam 
Just to let you know - With this update, we will be updating the notification and feed page for all the kind of activities we have already implemented from other tabs.

**Comment by avaidyam** [comment]
@sarithapillai8 That makes sense! Thanks for the heads up. As for the clarification, if a group is tapped, the streak pop-up should be saved for after ALL the activities in the group are completed, and THEN the user should be returned to the Feed tab. If there are multiple streak pop-ups then they should be presented one-after-the-other at this stage.

**Comment by sarithapillai8** [comment]
@avaidyam 

Functionalities completed with this task :
1) Moved games and group activities to assess tab
2) Added icon option for group activity settings
3) Study based activity selection in group activity creation - which was missing
4) Activities except games and survey were missing from feed and notification which are added now - including DBT, journal, scratch image, tip , and breathe.

Steak popup showing one by one after whole group activity execution was complex and confusing, so implemented like showing the steak after submitting the button in the same page for the group activity. Please check and let me know if it is fine.

**Comment by sarithapillai8** [comment]
@avaidyam 
Could you please suggest the action we need to do, if we have network issues in between the activity?.

**Comment by avaidyam** [comment]
@sarithapillai8 Please cache the ActivitySpec executable files (base64 HTML) before starting the group, and then submit all the ActivityEvents from the activities all at once at the very end. (The timestamps should still be separate though). We will later handle the network issues when handling offline mode as part of the native apps.

**Comment by sarithapillai8** [comment]
@avaidyam 
We don't have base64 for all the activities now. I mean we have it as a part of dashboard code now. So after converting that, we can  do these changes. But let me know if I need to change the current implementation of submitting the result one by one. Then  I think it will be better we also update the steak popup so that user can distinguish the activity for which it is showing.

**Comment by avaidyam** [comment]
Yes, please convert the current implementation to submitting all events at the very end, and if it's a quick change, then you can update the streak popups to match correctly. I've created a new issue #60 for the migration of the `LAMP-dashboard`-internal code into modules as part of `LAMP-activities`.

**Comment by sarithapillai8** [comment]
@avaidyam 
Implementing it like submitting at end will be easy if we are doing it from LAMP-activities. Is it ok to do this as part of #60, otherwise I will need to rework again while conversion ?

**Comment by avaidyam** [comment]
@sarithapillai8 I don't think I quite follow? 
> will be easy if we are doing it from LAMP-activities

The `LAMP-activities` code should not be submitting any data? It should be passed to the dashboard code which will then submit the data instead. But yes, batch-submitting data can be part of #60.

**Comment by sarithapillai8** [comment]
@avaidyam 
Yes, Lamp-activities submit result back to dashboard for saving events. The current implementation of group activity is like reusing the component which includes event submission in the component itself. If each activity need to be converted as base64 using LAMP-activities, it will be easy to implement this workflow after conversion.

**Comment by avaidyam** [comment]
@sarithapillai8 Ah I see what you mean -- that makes perfect sense. Thanks for the clarification!

**Comment by avaidyam** [comment]
@sarithapillai8 This doesn't work for me when testing using "Try It":
```
TypeError: undefined is not an object (evaluating 'e.settings.length')
```
Furthermore, I see a blank screen when I DO manage to open the group in the Assess tab.
